query-node/joystream-query-node/mappings/members.ts

```diff
@@ -1,3 +1,32 @@
-export function handleMemberRegistered() {
-	console.log(handleMemberRegistered.name);
+import { Memberships } from '../generated/indexer/entities/Memberships';
+import { DB } from '../generated/indexer';
+
+export async function handleMemberRegistered(db: DB) {
+  // Get event data
+  const { AccountId, MemberId } = db.event.event_params;
+
+  let member = new Memberships({ accountId: AccountId.toString(), memberId: +MemberId });
+
+  // Save to database.
+  db.save<Memberships>(member);
+
+  // Query from database
+  member = await db.get(Memberships, { where: { memberId: MemberId } });
+}
+
+export async function handleMemberUpdatedAboutText(db: DB) {
+  // Get event data
+  const { MemberId } = db.event.event_params;
+
+  // Query from database since it is an existsing user
+  const member = await db.get(Memberships, { where: { memberId: MemberId } });
+
+  // Make sure member exists
+  if (member) {
```

Comment:
Can you assert that `member` is defined, and not do an if branch that may ignore the failure, because if it isn't, then either the blockchain has a bug, or the query node processor has a bug, it is not a potentially valid event in this state.

Reply:
Done in https://github.com/Joystream/joystream/pull/418/commits/e35594ca09ca47d7a2eb93d8e7e4b6a1f8c21b91